.travis.yml

```diff
@@ -8,14 +8,16 @@ install:
 
 jobs:
   include:
-    - stage: test
-      script: make test
-    - stage: sonar scan
+    - stage: lint/test
```

Comment:
I combined them because the coverage report is generated from the tests and travis wont easily allow you to persist the reports between stages (annoying)  
https://docs.travis-ci.com/user/build-stages/#data-persistence-between-stages-and-jobs